cmd/cmd_controller.js

```diff
@@ -79,7 +79,8 @@ class EmbarkController {
       context: self.context,
       useDashboard: options.useDashboard,
       webServerConfig: webServerConfig,
-      webpackConfigName: options.webpackConfigName
+      webpackConfigName: options.webpackConfigName,
+      ipcRole: 'server'
```

Comment:
What would happen if two `embark run`s were run together?

Reply:
That doesn't seem to cause any problem:
run 1:
```
pid 96124 listening on /Users/anthony/code/embark-framework/embark/templates/demo/.embark/embark.ipc
========================
Welcome to Embark 3.2.0-develop
========================
ready to watch file changes
Starting Blockchain node in another process
IPFS node not found, attempting to start own node
Starting ipfs process
webserver available at http://localhost:8000
Blockchain node is ready
ipfs process started
Ethereum node detected..
loading solc compiler..
IPFS node detected..
compiling solidity contracts...
deploying contracts
deploying SimpleStorage with 134157 gas at the price of 1 Wei, estimated cost: 134157 Wei
SimpleStorage deployed at 0x1C233e6008b723E0766031B62D6E1e7Acf27546f using 134157 gas
finished deploying contracts
Deployment Done
running webpack with 'development' config...
writing file dist/js/dapp.js
writing file .embark/stats.report
writing file .embark/stats.json
writing file dist/index.html
Looking for documentation? You can find it at http://embark.status.im/docs/.
Ready
```

run 2:
```
pid 96211 listening on /Users/anthony/code/embark-framework/embark/templates/demo/.embark/embark.ipc
========================
Welcome to Embark 3.2.0-develop
========================
ready to watch file changes
Webserver already running on port 8000. Assigning an available port
webserver available at http://localhost:60239
Ethereum node detected..
loading solc compiler..
compiling solidity contracts...
deploying contracts
SimpleStorage already deployed at 0x1C233e6008b723E0766031B62D6E1e7Acf27546f
finished deploying contracts
Deployment Done
running webpack with 'development' config...
writing file dist/js/dapp.js
writing file .embark/stats.report
writing file .embark/stats.json
writing file dist/index.html
Looking for documentation? You can find it at http://embark.status.im/docs/.
Ready
```

Reply:
what about then running `embark test` ? which one does it connect to, will it get two responses?

Reply:
@iurimatias with a fresh `embark_demo` and double `embark run` (using this PR's branch, of course), I ran `embark test` in the same dir and got:

```
embark test
Compiling contracts


  SimpleStorage
    ✓ should set constructor value (17ms) - [0 gas]
    ✓ set storage value (90ms) - [26691 gas]
    ✓ should have account with balance (3ms) - [0 gas]


  3 passing (438ms) - [Total: 176948 gas]

 > All tests passed
```

Reply:
👍